src/EditorFeatures/Core/EditAndContinue/ActiveStatementTrackingService.cs

```diff
@@ -67,7 +74,8 @@ public async ValueTask StartTrackingAsync(Solution solution, IActiveStatementSpa
                 Contract.Fail("Can only track active statements for a single edit session.");
             }
 
-            await newSession.TrackActiveSpansAsync(solution, cancellationToken).ConfigureAwait(false);
+            var token = _listener.BeginAsyncOperation(nameof(ActiveStatementTrackingService));
+            _ = newSession.TrackActiveSpansAsync(solution).CompletesAsyncOperation(token);
```

Comment:
are there any implications on removing this await if we apply code updates before this is completed? e.g. we don't remap the leaf frame since we don't have the latest active statements information

Reply:
No, the tracker only affects the grey tags showing internal active statements in the editor and spans fed into source tree diff alg, but they are optional there.